getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+![Scuttlebutt Logo](assets/hermies-300x300.png)
+
+Ok, you've been hearing about [Scuttlebutt](https://scuttlebutt.nz)  and decided that "Yes, I do want to join an amazing social network with lots of good people that no company can control and also happens to also work offline. 
+
+Here's a quick overview with the basics you need to know.
+
+First off Scuttlebutt is a protocol on which many different kinds of apps can be built. 
+
+As for the social network, there are many clients, just like there are many Twitter clients. It doesn't really matter which one you use. They're all talking on the same network. 
+
+Patchwork is the most polished client and it doesn't require you to know any geeky stuff. The only caveat is that it just wants to run on one computer, and your identity is tied to the files on the computer you install it on. There are ways around this but it's beyond the scope of this.
```

Comment:
"The only caveat is that it just wants to run on one computer, and your identity is tied to the files on the computer you install it on. There are ways around this but it's beyond the scope of this." this is an opinion baked into the Scuttlebutt protocol, not Patchwork. as mentioned before it's not by accident. %7QPjGHfSvJMogZWDb1Mffbs7K9MwOt+INC8Z7A6V7Fc=.sha256

Reply:
so, would you suggest saying "The only caveat is scuttlebutt wants to only run on one computer..." ?  I think it's important to call out the only run it on one computer thing because that is _not_ normal and most people would expect to be able to run it on their work and home computers... and when we get phones they'll expect to be able to use the same identity there too (maybe with `sameAs` but still...).

Reply:
Scuttlebutt the protocol expects one key per uxer per device, so replication can be simple. if we don't already have a faq section on this, we should, since it's probably the most commonly asked question. however it doesn't mean you can't use Scuttlebutt apps on all your devices, `sameAs` will give us simple replication _and_ multi-device feeds.